docs/advanced/pool.rst

```diff
@@ -6,13 +6,13 @@ Connection pools
 ================
 
 A `connection pool`__ is an object managing a set of connections and allowing
-their use to functions needing one. Because the time to establish a new
+their use in functions needing one. Because the time to establish a new
 connection can be relatively long, keeping connections open can reduce the
-latency of a program operations.
+latency of program operations.
 
 .. __: https://en.wikipedia.org/wiki/Connection_pool
 
-This page explains a few basic concepts of Psycopg connection pool's
+This page explains a few basic concepts of Psycopg connection pools
```

Comment:
The previous version was more grammatically correct.

Reply:
OK